descope/tests/mocks/mgmt/managementmock_test.go

```diff
@@ -0,0 +1,13 @@
+package mgmtmocks
+
+import (
+	"testing"
+
+	"github.com/descope/go-sdk/descope/mgmt"
+)
+
+func TestManagementMock(t *testing.T) {
```

Comment:
don't understand this

Reply:
A test to check compilation, so if someone add new method they will need to add it to the mock as well